assemble/conf/log4j-monitor.properties

```diff
@@ -36,6 +36,17 @@ log4j.appender.GUI=org.apache.accumulo.server.monitor.LogService
 log4j.appender.GUI.Keep=50
 log4j.appender.GUI.Threshold=WARN
 
+## Configures Audit logs which are OFF by default.
+#log4j.appender.AUDIT=org.apache.log4j.DailyRollingFileAppender
+#log4j.appender.AUDIT.File=${accumulo.log.dir}/${accumulo.local.hostname}.audit
+#log4j.appender.AUDIT.DatePattern='.'yyyy-MM-dd
+#log4j.appender.AUDIT.layout=org.apache.log4j.PatternLayout
+#log4j.appender.AUDIT.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss,SSS/Z} [%c{2}] %-5p: %m%n
+#log4j.logger.org.apache.accumulo.audit=INFO, AUDIT
+#log4j.additivity.org.apache.accumulo.audit=false
+## Uncomment above and comment out line below to turn Audit logging ON
+log4j.logger.org.apache.accumulo.audit=OFF
```

Comment:
Should this be included in log4j-monitor.properties? I would have only expected to see it in log4j-service.properties.